src/tests/ie_test_utils/functional_test_utils/src/summary/api_summary.cpp

```diff
@@ -69,12 +69,13 @@ void ApiSummary::updateStat(ov_entity entity, const std::string& target_device,
     if (isCrashReported) {
         cur_stat[real_device].crashed--;
         isCrashReported = false;
+    } else {
```

Comment:
we need to report cur_stat[real_device].rel_all += rel_influence_coef; for all cases. take a look op _report

Reply:
Agree :) forget my code :D